en/12_Project_Governance/08_Fixed_dependencies.md

```diff
@@ -17,10 +17,10 @@ Dependency | Fixed major release line
 [`composer/installers`](https://packagist.org/packages/composer/installers) | 2
 [`guzzlehttp/guzzle`](https://packagist.org/packages/guzzlehttp/guzzle) | 7
 [`guzzlehttp/psr7`](https://packagist.org/packages/guzzlehttp/psr7) | 2
-[`intervention/image`](https://packagist.org/packages/intervention/image) | 2
+[`intervention/image`](https://packagist.org/packages/intervention/image) | 3
```

Comment:
Unrelated though missed updating this at the time

Reply:
The commit message is literally "Update fixed dependencies" - this is explicitly related :p